tests/test_main.py

```diff
@@ -21,12 +23,13 @@
 @pytest.mark.parametrize(
     ('cli_args', 'build_args'),
     [
-        ([], [cwd, out, ['sdist', 'wheel'], False]),
-        (['-s'], [cwd, out, ['sdist'], False]),
-        (['-w'], [cwd, out, ['wheel'], False]),
-        (['source'], ['source', out, ['sdist', 'wheel'], False]),
-        (['-o', 'out'], [cwd, 'out', ['sdist', 'wheel'], False]),
-        (['-x'], [cwd, out, ['sdist', 'wheel'], True]),
+        ([],            [cwd, out, ['sdist', 'wheel'], True, False]),
+        (['-n'],        [cwd, out, ['sdist', 'wheel'], False, False]),
+        (['-s'],        [cwd, out, ['sdist'], True, False]),
+        (['-w'],        [cwd, out, ['wheel'], True, False]),
+        (['source'],    ['source', out, ['sdist', 'wheel'], True, False]),
```

Comment:
Not sure I like this indentation technique. Looks ok now, but as soon as we add another line that's longer than the current longest we have to reindent all other lines which feel to me carries more risk than worth it (more chance of merge conflict, more lines changed than needed and potentially doing unintended changes).

Reply:
That is fair. We can put them on different lines, what do you think?

```python
(
    ['-n'],
    [cwd, out, ['sdist', 'wheel'], False, False],
)
```

Reply:
I'm happy to keep it one line which is what black also would do as long as it fits in one line.